### PR TITLE
Adding code of conduct file

### PR DESCRIPTION
**Pull request by Kerveyfelix**
Fixes issue #26 
Please review the PR.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at harshkumar9.in@gmail.com. All
```
Comment (plazzy99) [inline CODE_OF_CONDUCT.md:58]: correct the email id

Comment (plazzy99) [inline CODE_OF_CONDUCT.md:58]: instead of project team mention project maintainer.

**Review by plazzy99 — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Kerveyfelix) [reply CODE_OF_CONDUCT.md:58]: Yes. I will.

Comment (Kerveyfelix) [reply CODE_OF_CONDUCT.md:58]: Noted.

Comment (Kerveyfelix) [reply CODE_OF_CONDUCT.md]: I have made a new commit after making the required changes.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at harshkumar9.in@gmail.com. All
+reported by contacting the project maintainer at vatsalkesarwani12@gmail.com. All
 complaints will be reviewed and investigated and will result in a response that
 is deemed necessary and appropriate to the circumstances. The project team is
```
Comment (plazzy99) [inline CODE_OF_CONDUCT.md:60]: this line also contain project team

**Review by plazzy99 — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (Kerveyfelix) [reply CODE_OF_CONDUCT.md]: Making changes.

**Review by plazzy99 — APPROVED** [review]

**Pull request merged**